src/main/java/kr/ac/pknu/capstone/domain/Data/DataRepository.java

```diff
@@ -2,29 +2,31 @@
 
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.core.io.ClassPathResource;
 import org.springframework.stereotype.Repository;
 
 import java.io.File;
 import java.io.FileWriter;
 import java.io.IOException;
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Repository
 public class DataRepository {
 
     private final ObjectMapper objectMapper;
     private final List<Data> data;
 
+    @Autowired
     public DataRepository(ObjectMapper objectMapper) throws IOException {
         this.objectMapper = objectMapper;
         this.data = readJsonData();
     }
 
     private List<Data> readJsonData() throws IOException {
         ClassPathResource resource = new ClassPathResource("data.json");
-        File file = resource.getFile();
-        return objectMapper.readValue(file, new TypeReference<List<Data>>() {});
+        return objectMapper.readValue(resource.getInputStream(), new TypeReference<List<Data>>() {});
```

Comment:
빌드 후 jar 파일 실행 시 문제가 발생해서 수정한 부분이에요!
File을 이용하면 빌드 후 jar 파일 실행 시 오류를 뿐더라구요.
<details>
<summary>오류 코드</summary>
Caused by: java.io.FileNotFoundException: class path resource [data.json] cannot be resolved to absolute file path because it does not reside in the file system: jar:file:/C:/Users/shjy9/Projects/capstone/capstone/build/libs/capstone-0.0.1-SNAPSHOT.jar!/BOOT-INF/classes!/data.json
</details>

[참고](https://leegicheol.github.io/java/FileInputStream-error/)

Reply:
ㅋㅋㅋ 이상하네요 `inputStream` 으로 데이터를 읽는데 파일은 못찾는건 시스템 디버깅을 해야 할 듯 하네요 😭
이 부분은 확인했습니다!